### ISSUE TITLE
{% if object.allow_comments %} prevents form from rendering

### ISSUE THREAD

**Issue by boryskuczkowski** [danirus/django-comments-xtd#108]
Hi there, 

I was trying to resolve that issue, without any luck. The same issue was mentioned in stackoverflow here:
[https://stackoverflow.com/questions/51860222/comment-form-doesnt-load-when-using-django-comments-xtd](url)
So basically despite being authorised (logged in) the form doesn't display, the only way to display form is to strip it of `{% if object.allow_comments %}` tags. 
Anyone know how to address this in django 2.1?

**Comment by danirus** [comment]
Hi @boryskuczkowski. The evaluation of the condition `if object.allow_comments` will be `True` only if the attribute `allow_comments` does exist in the `object`. If the object you want to attach comments to doesn't have such attribute then the condition will always be `False`.

Look at the example sites of this app. The `Article` model [contains an attribute](https://github.com/danirus/django-comments-xtd/blob/master/example/simple/articles/models.py#L29) called `allow_comments`.

In other words, that attribute is not needed at all in the objects you want to add comments to. You do perfectly fine removing the condition `{% if object.allow_comments %}` from the template if the `object` doesn't have the attribute `allow_comment`.

**Comment by The-Phi** [comment]
Indeed, it was not very clear in the tutorial why this 'allow_comments' appears. I thought it was some sort of foreign key added by the app...
Maybe a little word on this in the tutorial could help....

**Comment by danirus** [comment]
Fixed in 8ddb672.
